CP1404/prac_05/emails.py

```diff
@@ -0,0 +1,18 @@
+user_completed = False
+
+emails = {}
+
+while not user_completed:
+    email = input("Email: ")
+    if email == "":
```

Comment:
There's no reason to use this pattern, when the standard one is better.
Don't use `continue` to exit a loop; it's for continuing to the next iteration.